kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -1,24 +1,81 @@
 <template>
 
-  <div class="roster">
-    <div>
-      <user-create-modal></user-create-modal>
-    </div>
-    <div class="user-roster">
-      <ul>
-        <li v-for="user in getUsers">
-          <div>{{ user.last_name + ", " + user.first_name + ", "}} {{user.roles.length ? user.roles[0].kind : "user" }}</div>
-          <user-edit-modal :userid="user.id" :roles="user.roles" :username="user.username" :firstname="user.first_name" :lastname="user.last_name"></user-edit-modal>
-          <button @click="deleteUser(user.id)">Delete</button>
-        </li>
-      </ul>
+  <div class="user-roster">
+
+    <h2>
+      {{userPageTitle}}
+    </h2>
+
+    <span> ( {{ visibleUsers.length }} )</span>
+
+    <div class="toolbar">
+      <select v-model="roleFilter" name="user-filter">
+        <option selected value="all"> All Users </option>
+        <option value="admin"> Admins </option>
+        <option value="learner"> Learners </option>
+      </select>
+
+    <div class="searchbar">
+        <svg class="search-button" src="../icons/search.svg"></svg>
```

Comment:
This should be a proper `<button>` with the `svg` inside - see `search-button.vue` for details.